org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/WDrillReport.java

```diff
@@ -0,0 +1,541 @@
+		this.setClosable(true);
+		ZKUpdateUtil.setWidth(this, "60%");
+		ZKUpdateUtil.setHeight(this, "70%");
+		this.setAttribute(Window.MODE_KEY, Window.MODE_EMBEDDED);
+		this.setSclass("help-window");
+
+		Borderlayout borderlayout = new Borderlayout();
+		this.appendChild(borderlayout);
+		ZKUpdateUtil.setHflex(borderlayout, "1");
+		ZKUpdateUtil.setVflex(borderlayout, "1");
+
+		Center centerPane = new Center();
+		centerPane.setAutoscroll(true);
+		borderlayout.appendChild(centerPane);
+
+		Div div = new Div();
+		centerPane.appendChild(div);
+		div.setStyle("width: 100%; height: 100%; overflow: auto;");
+		Table table = new Table();
+		table.setStyle("width: 100%;");
+		div.appendChild(table);
+
+		table.setWidgetAttribute("cellspacing", "0");
+		table.setWidgetAttribute("cellpadding", "0");
+		table.setWidgetAttribute("border", "0");
+
+		Tr tr = new Tr();
+		table.appendChild(tr);
+
+		Td td = new Td();
+		tr.appendChild(td);
+		td.appendChild(getHeader());
+
+		tr = new Tr();
+		table.appendChild(tr);
+
+		td = new Td();
+		tr.appendChild(td);
+		td.appendChild(getContent());
+	}
+
+	private Table getHeader()
+	{
+		Table table = new Table();
+		table.setWidgetAttribute("cellspacing", "0");
+		table.setWidgetAttribute("cellpadding", "0");
+		table.setWidgetAttribute("border", "0");
+		table.setStyle("width: 100%;");
+		table.setWidgetAttribute("class","help-window-header");
+
+		Tr tr = new Tr();
+		table.appendChild(tr);
+
+		Td td = new Td();
+		td.setWidgetAttribute("class","help-window-title");
+		tr.appendChild(td);
+		// generate a unique win prefix id for this help window
+		Calendar cal = Calendar.getInstance();
+		SimpleDateFormat sdf = new SimpleDateFormat("ddHHmmss");
+		String dt = sdf.format(cal.getTime());
+		winpref = "Win"+dt;
+		A a = new A();
+		a.setWidgetAttribute("name", winpref);
+		td.appendChild(a);
+		StringBuilder title = new StringBuilder(Msg.getMsg(Env.getCtx(), "DrillColumn")).append(": ").append(drillReportCtl.getTableName())
+				.append(" - ").append(!Util.isEmpty(drillReportCtl.getDisplayValue()) ? drillReportCtl.getDisplayValue() : drillReportCtl.getValue());
+		H3 h3 = new H3();
+		h3.appendChild(new Text(title.toString()));
+		td.appendChild(h3);
+
+
+		tr = new Tr();
+		table.appendChild(tr);
+
+		td = new Td();
+		tr.appendChild(td);
+
+		tr = new Tr();
+		table.appendChild(tr);
+
+		td = new Td();
+		tr.setWidgetAttribute("class","help-window-tabs");
+		tr.appendChild(td);
+
+		tr = new Tr();
+		table.appendChild(tr);
+
+		td = new Td();
+		tr.appendChild(td);
+
+		return table;
+	}
+
+	private Tabbox getContent()
+	{
+
+		Tabbox tabbox = new Tabbox();
+		Tabs tabs = new Tabs();
+		tabbox.appendChild(tabs);
+		Tabpanels tabpanels = new Tabpanels();
+		tabbox.appendChild(tabpanels);
+
+		Tab tab = new Tab(Msg.translate(Env.getCtx(), "AD_Process_DrillRule_ID"));
+		tabs.appendChild(tab);
+		tab = new Tab(Msg.translate(Env.getCtx(), "AD_Table_ID"));
```

Comment:
For translation from AD_Element, it is more efficient to use Msg.getElement(ctx, element) instead.